src/transformers/modeling_utils.py

```diff
@@ -2385,9 +2386,15 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
             modules_to_not_convert.extend(keep_in_fp32_modules)
 
             model = replace_8bit_linear(
-                model, threshold=load_in_8bit_threshold, modules_to_not_convert=modules_to_not_convert
+                model,
+                threshold=load_in_8bit_threshold,
+                modules_to_not_convert=modules_to_not_convert,
             )
 
+            model._is_int8_training_enabled = (
+                is_bitsandbytes_greater_0_37_0()
+            )  # training in 8-bit is only available in 0.37.0+
+
```

Comment:
Let's do the version check here and put the comment above the code to avoid this formatting.